RevenueCatUI/Templates/V2/ViewModelHelpers/PaywallComponentViewModel.swift

```diff
@@ -37,7 +37,7 @@ enum PaywallComponentViewModel {
     case tabControlToggle(TabControlToggleComponentViewModel)
 
     case carousel(CarouselComponentViewModel)
-
+    // case video(VideoComponentViewModel) to do: Add view model here
```

Comment:
I mentioned in this in another comment but since this enums are so tightly coupled, I would normally add an nearly empty view model in this PR for compile reasons and then build out the functionality in a follow up PR 🤷‍♂️